.changeset/empty-pillows-hunt.md

```diff
@@ -2,4 +2,151 @@
 '@primer/react': major
 ---
 
-Prepare library for `v35`
+### ActionList
+
```

Comment:
@mperrotti put the banner image from the technical previews in here, so they are part of the change log. Felt like a nice touch. If you want to do the same, you can use this:

![ActionList hero](https://user-images.githubusercontent.com/13340707/154946464-d1093087-a01c-49cd-bd55-1d2b60854f13.png)

Reply:
I like it! Will add them together in a different PR!